cluster/ci/config/prow/jenkins-config-updater.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: Template
+parameters:
+- name: SOURCE_URL
+  displayName: Source URL
+  description: Git source URL to use for building the image
+  value: https://github.com/openshift/test-infra.git
+- name: SOURCE_REF
+  displayName: Source ref
+  description: Git source ref to use for building the image
+  value: master
+objects:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: jenkins-config-updater
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: jenkins-config-updater-binaries
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: jenkins-config-updater:binaries
+    source:
+      git:
+        uri: "${SOURCE_URL}"
+        ref: "${SOURCE_REF}"
+      type: Git
+      dockerfile: |
+        FROM openshift/origin-release:golang-1.8
+
+        COPY . /go/src/k8s.io/test-infra
+        RUN cd /go/src/k8s.io/test-infra && \
+            godep restore && \
+            CGO_ENABLED=0 GOOS=linux go install -a --ldflags '-extldflags "-static"' ./experiment/jenkins-config-updater && \
+            rm -rf /go/src /go/pkg
+    strategy:
+      dockerStrategy:
+        from:
+          kind: DockerImage
+          name: openshift/origin-release:golang-1.8
+        imageOptimizationPolicy: SkipLayers
+    triggers:
+    - type: ConfigChange
+- apiVersion: v1
```

Comment:
@csrwng how can we base this buidlconfig off of `openshift/origin` and just add the `jenkins-config-updater` in? change the `git` source to a inline dockerfile that just copies?

Reply:
the Dockerfile can be inline or in the source repo ... doesn't matter.
It looks like it's already doing that: https://github.com/openshift/test-infra/pull/54/files#diff-250b79a38a6c04f921de7860c877896bR1
Just not based off of origin.

Reply:
Why do we need to base this off of `openshift/origin`?